src/Bridge/Doctrine/Orm/Filter/BooleanFilter.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Bridge\Doctrine\Orm\Filter;
+
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryNameGenerator;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\QueryBuilder;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Filter by a boolean value
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+class BooleanFilter extends AbstractFilter
+{
+    /**
+     * @var string Keyword used to retrieve the value.
+     */
+    private $booleanParameterName;
```

Comment:
Is in your exemple 'enabled' the field you want to check ?

I think it's better than what I proposed, it will be better to understand and to use.

Ill work on this & modify the test.

Thank you for the help,